File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.py
class Solution:
    def differenceOfSum(self, nums: List[int]) -> int:
        elemsum=0
        digitsum=0
        for num in nums:
            elemsum+=num
            while num!=0:
                dig=num%10
                digitsum+=dig
                num=num//10
        return elemsum-digitsum



        

File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.java
class Solution {
    public int differenceOfSum(int[] nums) {
        int elemsum=0;
        int digitsum=0;
        for (int num : nums){
            elemsum+=num;
            while (num!=0){
                int dig=num%10;
                digitsum+=dig;
                num=num/10;
            }}
        return elemsum-digitsum;

    }
}